library/hmac.po

```diff
@@ -57,28 +72,41 @@ msgid ""
 "The parameters *key*, *msg*, and *digest* have the same meaning as in :func:"
 "`~hmac.new`."
 msgstr ""
+"Renvoie l'empreinte cryptographique de *msg* pour une *key* secrète et un "
```

Comment:
Question : Mélange-on *empreinte cryptographique* et *condensat* juste pour éviter les répétitions, ou essaye-on d'uniformiser ? 
 
```suggestion
"Renvoie le condensat de *msg* pour une *key* secrète et un "
```

J'aurai tendance à préférer aujourd'hui *condensat*, plus court et pas moins explicite, comme proposé par @christopheNan dans https://github.com/python/python-docs-fr/pull/823#discussion_r294082753, en acceptant quand vraiment la répétition est trop lourde un petit "empreinte cryptographique" de temps en temps.

Attention, parfois *digest* se traduit *fonction de hachage* : parfois ils parlent du résultat, parfois ils parlent de l'algorithme qui le produit.

Typiquement dans la phrase :

    Return the digest of the bytes passed to the update() method so far. This bytes object will be the same length as the digest_size of the digest given to the constructor.

Le premier `digest` c'est le condensat, le 2nd `digest` c'est l'algorithme qui a produit le premier, donc une fonction de hachage.

Reply:
```suggestion
"Renvoie le condensat de *msg* correspondant à la clé secrète *key* et à l'algorithme "
```
L'article défini pour *clé secrète* me paraît important. Le reste est de la formulation.